### PR TITLE
chore: reconfigure renovate

### PR DESCRIPTION
**Pull request by viceice**
Add some regex managers to update the `Dockerfile`

### CODE REVIEW

**Comment by rarkins** [comment]
I would like to consider having a single manager and using comments *above* the env instead, what do you think?

Alternatively, I think these env vars are so common in Dockerfiles that we could make them presets and maybe even part of config:base

**Comment by viceice** [comment]
Yeah, working on it, will try it on docker composer image repo first

**Comment by viceice** [comment]
@rarkins Ready to review again.

**Comment by viceice** [comment]
same regex works for `renovatebot/composer` 💪

**Comment by rarkins** [comment]
Code suggestions is broken for me, so doing it as a comment here

Instead of `# renovate: github-releases/docker/docker-ce&versioning=docker`, do you think `# renovate: github-releases/docker/docker-ce versioning=docker` is cleaner? i.e. a space divider instead of `&`

**Comment by viceice** [comment]
> Code suggestions is broken for me, so doing it as a comment here
> 
> Instead of `# renovate: github-releases/docker/docker-ce&versioning=docker`, do you think `# renovate: github-releases/docker/docker-ce versioning=docker` is cleaner? i.e. a space divider instead of `&`

What about:
```Dockerfile
# renovate: datasource=github-releases name=docker/docker-ce versioning=docker
ENV DOCKER_VERSION=19.03.5
```
or
```Dockerfile
# renovate: datasource:github-releases name:docker/docker-ce versioning:docker
ENV DOCKER_VERSION=19.03.5
```

**Comment by rarkins** [comment]
I like this approach best: `# renovate: datasource=github-releases name=docker/docker-ce versioning=docker`

My only doubt is whether to use `depName` and optionally `lookupName` for consistency

**Comment by viceice** [comment]
🤔 Ok, we can use wharever we want as key, because it's only relevant for regex matching,  but the key will not be the capture group name by default. We would make it explicit. I'll try to update the regex.

**Comment by rarkins** [comment]
I think the simplest approach is to use the capture groups, and also plenty will probably copy from this repo's config to learn, so it can be good for consistency to have like `depName=(<?depName>.*?)`

**Comment by viceice** [comment]
see current changes

File: Dockerfile [hunk 1]
```diff
@@ -187,6 +193,7 @@ RUN pip install --user pipenv
 
 # Poetry
 
+# renovate: datasorce=github-releases depName=python-poetry/poetry
```
Comment (rarkins) [inline Dockerfile]:
```suggestion
# renovate: datasource=github-releases depName=python-poetry/poetry
```

**Comment by viceice** [comment]
Working, see #5728 💪

**Pull request merged**